pinot-segment-local/src/main/java/org/apache/pinot/segment/local/data/manager/TableDataManagerParams.java

```diff
@@ -56,6 +76,38 @@ public void setStreamSegmentDownloadUntarRateLimitBytesPerSec(long streamSegment
     _streamSegmentDownloadUntarRateLimitBytesPerSec = streamSegmentDownloadUntarRateLimitBytesPerSec;
   }
 
+  public boolean isRetrySegmentDownloadUntarFailure() {
+    return _isRetrySegmentDownloadUntarFailure;
+  }
+
+  public void setRetrySegmentDownloadUntarFailure(boolean retrySegmentDownloadUntarFailure) {
+    _isRetrySegmentDownloadUntarFailure = retrySegmentDownloadUntarFailure;
+  }
+
+  public int getRetryCount() {
```

Comment:
We need to rename these config keys to be specific for the segment download untar retry

Reply:
These variables have been renamed now